core/mono-samples/iOS/Program.csproj

```diff
@@ -6,13 +6,13 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Runtime.iOS.Sample.Mono" Version="6.0.0-dev" />
+    <PackageReference Include="Microsoft.NET.Runtime.iOS.Sample.Mono" Version="6.0.0-*" GeneratePathProperty="true"/>
```

Comment:
Looks like this doesn't work on CI because this repo installs the 5.0 dotnet sdk. I tried locally and what works is using `$(BundledNETCorePlatformsPackageVersion)` as the version since then it'll try to get the same version as it uses for the runtime pack.

That brings up another issue though, right now it tries to restore `Microsoft.NETCore.App.Runtime.ios-x64 with version (= 5.0.0)` but the final 5.0 release isn't published to nuget.org yet. And even if it were we don't really want to use the 5.0 version.

As far as I know the only way to consume 6.0 assets is by installing nightly master builds of the dotnet sdk from the dotnet/installer repo. I don't know if we want to do that in CI in this repo (maybe we just need to skip this project there).

Additionally we can add the following global.json next to Program.csproj to force usage of the 6.0 SDK:

```
{
  "sdk": {
    "version": "6.0.100-alpha",
    "rollForward": "major",
    "allowPrerelease": true
  }
}
```

Reply:
Yeah, I think skipping the project makes sense for now.  It's in a bit of limbo.

Reply:
@adegeo Tagging you because you made the last commit to the yml.  Can we add another leg that installs the net6 preview sdk and builds the iOS project?  That seems to be the only way to be correct on CI for us. Otherwise, we can just skip the project.

Reply:
@steveisok Sorry, missed this alert. Yeah we could do that easily as long as the dotnet-install powershell script supports it?

Reply:
@adegeo Looks like it does with the `--version` switch.  Would it be easier to bump the sdk version in CI or just add another leg?

Reply:
@steveisok If you can give me the exactly command line I'll edit the script to add a leg.

Reply:
Fixed the version at `6.0.100-alpha.1.20531.2`.